test/e2e/common.go

```diff
@@ -204,3 +206,23 @@ func updateKustomization(kustomizeDirPath string, kustomizeResourcePaths []strin
 
 	return nil
 }
+
+func parseCollectorJson(collectorOutput string) map[string]interface{} {
```

Comment:
Worth considering if declaring struct of fields we are interested in and unmarshalling json into it, would simplify our code.